### DISCUSSION TITLE
Non-FIDO member greetings (and test)

### DISCUSSION THREAD

**Discussion by dbrowski** [fido-alliance/discussions#1] [General]
Hello!

Anthony Dombrowski here. Looks like it allows me access and the ability to create a discussion!

**Comment by sandandsnow** [comment]
Thank you for setting this up.

**Comment by RyanSinger** [comment]
Hello!

**Comment by dturnerx** [comment]
What are your thoughts about this vs Google groups?

**Comment by RyanSinger** [comment]
It's a similar interface for me either way, as my preferred client is gmail.

--
Ryan Singer

**Reply by dbrowski** [reply]
Ya, if on GitHub, I can come here directly or reply through the email. While I can reply through the Google Groups interface, I rarely visit there. So, the GitHub discussions offer a little more easy access and flexibility for me.
